stdlib/public/core/Sort.swift.gyb

```diff
@@ -262,6 +310,7 @@ func _heapify<C>(
       ${", by: areInIncreasingOrder" if p else ""})
   }
 }
+
```

Comment:
Not that these are particularly bad, but we try to avoid whitespace-only changes, for the record.